frontend/src/Student/components/Shows.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import ShowCard from '../components/ShowCard'
+
+const Shows = ({shows, loading}) => (
+  <div>
+    {loading ? null : shows.map((show) => (
+      <ShowCard key={show.id} {...show} />
+    ))}
+  </div>
+)
+
+Shows.propTypes = {
+  shows: PropTypes.array,
```

Comment:
I'd make this required and add a `defaultProp` of `[]`

Reply:
If there are no so associated with the student, would they be able to still log in?